HelpSource/Tutorials/Streams-Patterns-Events4.schelp

```diff
@@ -156,7 +156,7 @@ Task({
 				Env.sine, doneAction: Done.freeSelf, timeScale: 1.0 + 6.0.rand, levelScale: 0.3
 			);
 			Out.ar(out, sound);
-		}).send(s);
+		}).add;
```

Comment:
keep send because we don't want to accumulate synthdefs, it is on purpose.

Reply:
Thank you, and my apologies for the change. I have reverted it in https://github.com/supercollider/supercollider/pull/7117/commits/af05e86003f4e48eebe42e01a1ab847b467b96e5